src/utils/string.js

```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+import {getRandomInts} from './random';
+
+const CHARS = '0123456789ABCDEF'.split('');
```

Comment:
Minor, but `split` is not needed here. Chars in a string are addressable by `[i]`